meson.build

```diff
@@ -0,0 +1,82 @@
+project('SU2', 'c', 'cpp', 
+        version: '6.2.0', 
+	license: 'LGPL2',
+	default_options: ['buildtype=release', 
+                          'warning_level=0', 
+                          'cpp_std=c++11'])
+
+
+pymod = import('python')
+python = pymod.find_installation()
+
+su2_cpp_args = ['-Wno-unused-parameter', '-Wno-empty-body', '-Wno-format-security']
```

Comment:
What do you thing of adding -Wno-deprecated-declarations to this list? (We get tons of that in the AD build)

Reply:
Thats a good idea. Let's do that.